migrations/sql/V2019.07.29.1316__InitialSchema.sql

```diff
@@ -7,3 +7,9 @@ CREATE TABLE users
   serializedprofile varchar(10000),
   PRIMARY KEY ("id")
 );
+
+CREATE TABLE onetime_token
+(
+  id serial primary key,
```

Comment:
Why have a numerical id? The tokens are presumably unique?

Reply:
This is how it is in Montagu, not sure why!

Reply:
Did tokens potentially get reused in Montagu? Agree shouldn't be any need for us to have this atm, will take out.

Reply:
I just dug around and found this explanation as to why we used a numerical id in Montagu! https://vimc.myjetbrains.com/youtrack/issue/VIMC-2087

I _think_ in this case we should be alright as we won't be encoding roles/permissions in the token

Reply:
Ah ok, makes sense. Seems to work ok from my testing so far...